buildSrc/src/main/kotlin/scripts/spotless.gradle.kts

```diff
@@ -0,0 +1,25 @@
+package scripts
+
+apply(plugin = "com.diffplug.spotless")
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        target("**/*.kt")
+        ktlint().userData(
+            mapOf(
+                "disabled_rules" to "import-ordering",
+                "max_line_length" to "150"
```

Comment:
Are we following `150` or `140` as it states in the [docs](https://wearezeta.atlassian.net/wiki/spaces/AR/pages/122520039/Code+Style+Guideline#Max-Line-Length)?

Reply:
This config it's overwriting that rule, I will adjust to what stated in that docs, good catch thanks :D